src/dnsmasq/forward.c

```diff
@@ -1813,6 +1813,13 @@ void receive_query(struct listener *listen, time_t now)
       
       if (m >= 1)
 	{
+	  if (stale && have_pseudoheader)
```

Comment:
This is a commit we imported from the `dnsmasq` repository, can you give us more details what you think might be wrong here?

Reply:
stale is undefined when i try to compile

Reply:
You are right. This was a mistake by the auto-merge during the imports of the patches from the `dnsmasq` repository. I fixed it and did a manual `diff` inspection to verify this didn't happen anywhere else, too. Our automatic toolchain hasn't found it as we are not compiling FTL with `conntrack` support by default to keep dependencies at a minimum.

Reply:
> as we are not compiling FTL with conntrack support by default to keep dependencies at a minimum

Should we have a test compiling FTL on one architecture with all available options? Just in case...

P.S. Does `dnsmasq` have some automatic build tests in their toolchain?